### PR TITLE
Fellowships : Financial Terms

### PR DESCRIPTION
**Pull request by gvn**
Related : #121

### CODE REVIEW

**Comment by gvn** [comment]
One more of the tabs built out.

This needs [this Mozmaker PR](https://github.com/mozilla/mozmaker/pull/66) to look right.

File: app/react/pages/fellowships/fellowships.jsx [hunk 1]
```diff
@@ -58,7 +58,121 @@ export default React.createClass({
                 <IconItem imgSrc="/assets/img/fellowships/icon-open.svg" copy="Have experience participating in open communities."></IconItem>
               </div>
             </div>
-            <div className="p-y-3" name="Financial Terms">Schedule content.</div>
+            <div className="p-y-3" name="Financial Terms">
+              <p><em>{`To The Fellowships are designed to allow for flexibility for both Fellows and their families. The fellowship offers a stipend of USD$60,000 paid in 10 monthly installments. Fellows are responsible for remitting all applicable taxes and other government payments required.`}</em></p>
```
Comment (alanmoo) [inline app/react/pages/fellowships/fellowships.jsx]: Gremlin

**Comment by alanmoo** [comment]
All the `col-sm-6` classNames also need `col-xs-12`

**Comment by alanmoo** [comment]
Really close

**Comment by gvn** [comment]
Well, apparently this tab doesn't actually exist anymore, so that was fun...

**Pull request closed without merging**